docs/src/updated_example.md

```diff
@@ -0,0 +1,111 @@
+# Uprading docs from Documenter.jl to DocumenterVitepress.jl
+
+Assuming that your current documentation is based on Documenter.jl, you can upgrade to DocumenterVitepress.jl by following these steps:
+
+1. Install DocumenterVitepress.jl using the following julia command in your package's environment:
+
+```julia
+using Pkg
+Pkg.add("DocumenterVitepress")
+```
+
+2. Now go the the `make.jl` file in your `docs` folder and do the following necessary changes to upgrade to DocumenterVitepress.jl :
+
+
+   a. The `make.jl` file with `Documenter,jl` look like this:
+
+```julia
+using OMOPCDMPathways
```

Comment:
maybe consider an easier package's name? perhaps `using Example`?

Reply:
Actually this is the actual package that we are currently developing here-https://github.com/JuliaHealth/OMOPCDMPathways.jl

Reply:
As a documentation example it's easier for the user to have something familiar ;)